site/packages/aa-core/smart-account-client/types/userOperationOverrides.md

```diff
@@ -51,15 +51,15 @@ export type UserOperationOverrides = Partial<{
 
 ```ts [user-operation-override.ts]
 import type { UserOperationOverrides } from "@alchemy/aa-core";
-import { provider } from "./provider.ts";
+import { smartAccountClient } from "./smartAccountClient.ts";
 
 // Find your Gas Manager policy id at:
 //dashboard.alchemy.com/gas-manager/policy/create
 const GAS_MANAGER_POLICY_ID = "YourGasManagerPolicyId";
 
 // Link the provider with the Gas Manager. This ensures user operations
 // sent with this provider get sponsorship from the Gas Manager.
-provider.withAlchemyGasManager({
+smartAccountClient.withAlchemyGasManager({
```

Comment:
Geez there were a lot of snippets directly in markdown. There should be a snippet in snippets/aa-alchemy that creates a gas manager enabled client. If not in there then in snippets/sponsoring gas

We should swap the code out here with that snippet instead